package.json

```diff
@@ -14,7 +14,8 @@
   },
   "scripts": {
     "test": "xo && mocha",
-    "bench": "matcha bench.js"
+    "bench": "matcha bench.js",
+    "jshint": "jshint index.js"
```

Comment:
Yes, but it requires that it be installed globally and throws an exception when running `npm test`. Added it to the package.json so that it will be installed.